README.md

```diff
@@ -8,6 +8,16 @@
 
 The native RADOS library and development headers are expected to be installed.
 
+On debian:
+```sh
+sudo apt install -y libcephfs-dev librbd-dev librados-dev
+```
+
+On rhel:
```

Comment:
`nitpick` i woul add wrote : 
on RPMs based OS. this include Fedora/Centos/Opensuse  and also the Linux Entreprise based.
Rhel is something that not all people have.

Reply:
I'll play naive, but cant debian systems also install rpms? - is there a more accurate terminology that should be used to describe the red hat derivatives?

I am 100% fine with changing the name.